eva-server/src/test/java/uk/ac/ebi/eva/server/ws/RegionWSServerTest.java

```diff
@@ -51,6 +52,19 @@ public void testGetVariantsByRegions() throws URISyntaxException {
         testGetVariantsByRegionHelper("20:60000-61000,20:61500-62500", 17);
     }
 
+    @Test
+    public void testExcludeNested() throws URISyntaxException {
+        Response response = given().param("species", "mmusculus_grcm38").param("exclude", "filesAttrs").get(new URI("/v1/segments/20:60000-62000/variants"));
```

Comment:
The key should be `sourceEntries.attributes` to match the names returned by the web service.